generator/README.md

```diff
@@ -22,6 +22,23 @@ Generates a JSON file representing the in-memory decision tree.  This is a utili
 
 Generates a [DOT](https://en.wikipedia.org/wiki/DOT_(graph_description_language))-compliant representation of the decision tree, for manual inspection.
 
+To visualise the tree, you can also use [Graphviz](https://www.graphviz.org/). 
+
+Install the tool and create this file on your local environment editting the path to your installation of the tool:
```

Comment:
This is a windows only implementation, i would omit this section (and the .bat file) and state that the tool can be used to open the visualised tree (.gv) files.

This makes it platform agnostic, i.e. if there is a unix version out there it can be used also.

Reply:
Also, you could mention how graphviz has been proven to work, other visualisers (e.g. the web version you tried the other day) may have issues.

The requirements for a visualiser are known (currently) as:
* .gv files are encoded with UTF-8, visualisers must support this encoding.
* .gv files can include HTML encoded entities, visualisers should support this feature

Reply:
updated.